Sources/TuistKit/Generator/ConfigGenerator.swift

```diff
@@ -165,6 +170,17 @@ final class ConfigGenerator: ConfigGenerating {
             settings["MACH_O_TYPE"] = "staticlib"
         }
 
+        if target.product.isTest {
+            let appDependencies = graph.targetDependencies(path: sourceRootPath, name: target.name).filter { targetNode in
```

Comment:
Nitpick: You can do 

```swift
let appDependency = find(where: { $0.targetNode.target.product == .app }
```

Reply:
Where is the `find` function? I don't think it's defined.

Reply:
`first(where:` 🙂